vuepress/docs/v7.0/docs/curate/ecr-troubleshooting-guide.md

```diff
@@ -1,70 +1,65 @@
+3. Then proceed with the new installation.
 
-If you're actively working on your bundle and you simply want to verify things are working correctly, instead of generating a new tag for each release of the bundle you can try to keep overriding a specific tag using the git command `git tag -f`. We suggest you to follow this practice only during development and not in production.
+Note: If you're actively working on your bundle and simply want to verify that things are working correctly, you can override a specific tag using the git command `git tag -f` instead of generating a new tag for each release of the bundle. You should follow this practice only during development, never in production.
 
-## Bundle installation fails because a file has not been found in the bundle
+## 3. ERROR - File not found in bundle
+**Installation fails because a file has not been found in the bundle**
 
-### Overview
-When a component that is referenced in the `descriptor.yaml` is missing in the bundle or not correctly referenced, the bundle installation fails and in the logs is reported which file has not been found.
+When a component that is referenced in the `descriptor.yaml` is missing in the bundle or not properly called, the bundle installation will fail and the error reported in the logs.
 
 ```
 ERROR - File with name {filename} not found in the bundle
 ```
 
 ### Solution
 
-When such a problem happens, verify that the component referenced in the descriptor file are actually present in the bundle and that the reference is properly typed.
+Verify that the component named in the descriptor file is actually present in the bundle, in the location specified, and that the reference is properly formatted.
 
 Publish a new version of your bundle as described in the 
-["My bundle has an issue"](#my-bundle-has-an-issue-how-should-i-publish-a-new-version-of-the-bundle) section
+["How do I publish a new version"](#how-do-i-publish-a-new-bundle-version) section.
 
-## Bundle installation failed due to plugin(s) images not reachable
+## 4. My plugin Docker image is unreachable 
+**Bundle installation fails due to plugin(s) images that are not reachable**
 
-### Overview
-Plugin included in a bundle are referenced using their docker image. Sometime the image is not available - maybe has not yet be published or is in a private docker registry - and plugin installation can't happen and the entire bundle installation process can't finish successfully
+A bundle installation does not complete successfully because a plugin in a bundle, defined by a Docker image, in not available. 
 
 ### Solution
+This may happen if the Docker image for the plugin was not yet published or located in a private registry. Verify that the Docker image you are referencing is published, correctly formatted, and publicly available.
```

Comment:
can read as "this may happen if the image was not located in a private repo" -> "...published or is located..." (add "is")